config/routes.rb

```diff
@@ -2,4 +2,18 @@
   devise_for :users
   root to: 'pages#home'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+
+  namespace :coach do
+    resources :lessons
+  end
+
+  resources :sports, only: [:show] do
```

Comment:
On the sports page, we want to show all the sports right ?  So should it be index instead of show?